src/lsm/manifest_level.zig

```diff
@@ -86,7 +86,11 @@ pub fn ManifestLevelType(
         /// * The table is mutable so that this function can update its snapshot.
         /// * Asserts that the table currently has snapshot_max of math.maxInt(u64).
         /// * Asserts that the table exists in the manifest.
-        pub fn set_snapshot_max(level: *Self, snapshot: u64, table: *TableInfo) void {
+        pub fn set_snapshot_max(
+            level: *Self,
+            snapshot: u64,
+            table: *TableInfo,
+        ) void {
```

Comment:
This is still under 100 columns I think?

Why do we add the trailing comma after the `*TableInfo` arg, since there isn't a need to reflow right?

Reply:
whoops, artifact from old `TableInfo.flags` implementation